core/operation/src/main/java/uk/gov/gchq/gaffer/named/operation/NamedOperationDetail.java

```diff
@@ -58,15 +59,15 @@
     public NamedOperationDetail() {
     }
 
-    public NamedOperationDetail(final String operationName, final String description, final String userId,
+    public NamedOperationDetail(final String operationName, final String label, final String description, final String userId,
```

Comment:
For backwards compatability, these constructors should remain the same and instead call a new constructor which has the label as the final parameter.

So instead of adding a label in all the constructors:
```java
public Example(String example1, String Label, String example2) {
    this(example1, label, example2, null);
}

public Example(String example1, String Label, String example2, Integer example3) {
    // do stuff
}
```
Leave the constructors as they are so any existing code still call them and they maintain the same behaviour:
```java
public Example(String example1, String example2) {
    this(example1, example2, null, null);
}

public Example(String example1, String example2, Integer example3) {
    this(example1, example2, example3, null);
}

// new constructor
public Example(String example1, String example2, Integer example3, String label) {
    // do stuff
}
```

Reply:
Agreed, thanks for the recommended solution

Reply:
Done